Shared/Views/CampaignView.swift

```diff
@@ -146,6 +146,19 @@ struct CampaignView: View {
                     }
                     .id("Milestones")
                     
+                    if initialCampaign?.user.name == "Relay FM" {
+                        GroupBox {
+                            HStack(alignment: .top) {
+                                Image(systemName: "info.circle")
+                                    .padding(.top, 2)
+                                Text("These milestones are achieved when the overall fundraiser total reaches the specified amount, not this specific campaign.")
```

Comment:
Total might be extraneous. Feels like a lot of text.

Reply:
I'll merge this for now and think about wording a bit later!